aws/templates/application/application_lambda.ftl

```diff
@@ -306,6 +306,27 @@
                                 )
                             ) /]
                 [/#if]
+                [@cfScript
```

Comment:
Would we need to check that the lambda is attached to a VPC? Will the command fail if there are no interfaces found?

Reply:
@roleyfoley checks have been added